service-front/module/Application/src/Helpers/DTO/FormProcessorResponseDto.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Helpers\DTO;
+
+use Laminas\Form\FormInterface;
+
+class FormProcessorResponseDto
+{
+    public function __construct(
+        private string $uuid,
+        private FormInterface $form,
+        private array $responseData,
```

Comment:
I think you've removed the getter for this, but the property is still here and I don't think it's doing anything